src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,45 @@
+.icon {
+  display: block;
+  width: 24px;
+  height: 24px;
+  background-size: cover;
+
+  @include hover(transform, scale(1.2));
+
+  @include onTablet {
+    height: 32px;
+    width: 32px;
+  }
+
+  &--phone {
+    background-image: url(../images/icons/icon-phone.svg);
+
+    @include onDesktop {
+      &:hover::before {
+        position: absolute;
+        top: 50%;
+        left: -160px;
+        transform: translateY(-50%);
+
+        content: "+1 234 555-55-55";
```

Comment:
It would be better to keep such data in a variable.

Reply:
done